packages/lib-react-components/src/Markdownz/Markdownz.js

```diff
@@ -1,34 +1,152 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { Markdown as GrommetMarkdown } from 'grommet'
-import defaultOptions from './defaultOptions'
-
-const { createAnchorForNewTab, sanitizeForMarkdown } = defaultOptions
-
-export default function Markdownz (props) {
-  const { children, options } = props
-  const mergedOptions =  {
-    createElement(type, props, children) {
-      if (type === 'a' && props.href && props.href.startsWith('+tab+')) {
-        return createAnchorForNewTab(type, props, children)
-      }
-
-      return React.createElement(type, props, children)
-    },
-    ...options
+import {
+  Anchor,
+  Heading,
+  Image,
+  Paragraph,
+  Table,
+  TableBody,
+  TableCell,
+  TableRow,
+  TableHeader,
+  TableFooter,
+  Text,
+  Video
+} from 'grommet'
+
+import mime from 'mime-types'
+import remark from 'remark'
+import remark2react from 'remark-react'
+import emoji from 'remark-emoji'
+import remarkSubSuper from 'remark-sub-super'
+import externalLinks from 'remark-external-links'
+import toc from 'remark-toc'
+import ping from './lib/ping'
+
+const hashtag = '#'
+const at = '@'
+const subjectSymbol = '^S'
+
+class Markdownz extends React.Component {
+  buildResourceURL(resource, symbol) {
+    if (!resource) return ''
+    const { baseURI, projectSlug } = this.props
+    const baseURL = (projectSlug) ? `${baseURI}/projects/${projectSlug}` : baseURI
+
+    if (symbol === hashtag) return (projectSlug) ? `${baseURL}/talk/tag/${resource}` : `${baseURL}/talk/search?query=%23${resource}`
+    if (symbol === at) return `${baseURL}/users/${resource}`
+    if (symbol === subjectSymbol && projectSlug) return `${baseURL}/talk/subjects/${resource}`
+
+    return ''
+  }
+
+  shouldResourceBeLinkable(resource, symbol) {
+    const { projectSlug, restrictedUserNames } = this.props
+
+    if (symbol === at) return !restrictedUserNames.includes(resource)
+    if (symbol === subjectSymbol) return !!projectSlug
+
+    return true
+  }
+
+  // Support image resizing, video, and audio using markdown's image syntax
+  renderMedia(nodeProps) {
+    let altText, width, height
+    const imgSizeRegex = /=(\d+(%|px|em|rem|vw)?)x(\d+(%|px|em|rem|vh)?)/
+    let alt = nodeProps.alt
+    const src = nodeProps.src
+    const match = alt.match(imgSizeRegex)
+
+    if (match && match.length > 0) {
+      width = match[1]
+      height = match[3]
+      altText = alt.split(match[0])[0].trim()
+    }
+    const mimeType = mime.lookup(src)
+
+    if (mimeType && mimeType.includes('video')) {
+      return <Video a11yTitle={alt} controls="below" preload='metadata' src={src}><Paragraph>This video file is not supported</Paragraph></Video>
+    }
+    if (mimeType && mimeType.includes('audio')) {
+      // Grommet doesn't have an audio component right now
+      return <audio aria-title={alt} controls preload='metadata' src={src}><Paragraph>This audio file is not supported</Paragraph></audio>
```

Comment:
VoiceOver reads the audio element as 'audio group' in the storybook. I think `aria-title` here should be [`aria-label`](https://www.w3.org/TR/wai-aria/#aria-label).